x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/bulk_actions/translations.tsx

```diff
@@ -55,3 +55,42 @@ export const bulkApplyTimelineTemplate = {
     />
   ),
 };
+
+export const bulkAddRuleActions = {
+  FORM_TITLE: i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.addRuleActions.formTitle',
+    {
+      defaultMessage: 'Add rule actions',
+    }
+  ),
+
+  OVERWRITE_LABEL: i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.addRuleActions.overwriteCheckboxLabel',
+    {
+      defaultMessage: 'Overwrite all selected rules actions',
+    }
+  ),
+
+  THROTTLE_LABEL: i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.addRuleActions.throttleLabel',
+    {
+      defaultMessage: 'Actions frequency',
+    }
+  ),
+
+  THROTTLE_HELP_TEXT: i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.addRuleActions.throttleHelpText',
+    {
+      defaultMessage:
+        'Select when automated actions should be performed if a rule evaluates as true.',
+    }
+  ),
+
+  RULE_VARIABLES_DETAIL: i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.addRuleActions.ruleVariablesDetail',
+    {
+      defaultMessage:
+        'Rule variables may only affect some of the rules that you selected based on the rule types(for example \\u007b\\u007bcontext.rule.threshold\\u007d\\u007d will only work for threshold rule type).',
```

Comment:
@joepeeples 
> For example, for non-threshold type rules, will {{context.rule.threshold}} appear as literal text? or empty space? or error/fail?

It will appear as empty space

Reply:
Thanks @vitaliidm — how about "will only display values..." instead of "will only work..."?